src/test/java/edu/kis/powp/jobs2d/events/SelectTestFigureOptionListener.java

```diff
@@ -6,16 +6,28 @@
 import edu.kis.powp.jobs2d.drivers.DriverManager;
 import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
 
-public class SelectTestFigureOptionListener implements ActionListener {
+public class SelectTestFigureOptionListener implements ActionListener
+{
 
 	private DriverManager driverManager;
 
-	public SelectTestFigureOptionListener(DriverManager driverManager) {
+	public SelectTestFigureOptionListener(DriverManager driverManager)
+	{
 		this.driverManager = driverManager;
 	}
 
 	@Override
-	public void actionPerformed(ActionEvent e) {
-		FiguresJoe.figureScript1(driverManager.getCurrentDriver());
-	}
+	public void actionPerformed(ActionEvent e)
+	{
+		//FiguresJoe.figureScript1(driverManager.getCurrentDriver()); now we have 2 Joe's
+
+        if (e.getActionCommand().compareTo("Figure Joe1") == 0)
```

Comment:
Menu action should not depend on its caption. Enum or separate listeners would be better. At least you should consider constants.